packages/hooks/src/useAuthorize.ts

```diff
@@ -8,7 +8,10 @@ const useAuthorize = (token?: string) => {
     const { accounts, loginid = '' } = client;
     const current_token = accounts[loginid || ''].token;
 
-    const { data, ...rest } = useFetch('authorize', { payload: { authorize: token || current_token } });
+    const { data, ...rest } = useFetch('authorize', {
+        payload: { authorize: token || current_token },
+        options: { keepPreviousData: true },
+    });
```

Comment:
@sergei-deriv Please refer [this](https://github.com/binary-com/deriv-app/pull/9362/files#r1264304986).
```suggestion
     const { data, ...rest } = useFetch('authorize', { payload: { authorize: token || current_token } });
```